docassemble/ALWeaver/data/questions/review_screen.yml

```diff
@@ -0,0 +1,486 @@
+  Edit your answers about Weaverdata
+subquestion: |-
+  ${ weaverdata.table }
+---
+event: |-
+  jurisdiction_choices.revisit
+id: |-
+  revisit jurisdiction_choices
+question: |-
+  Edit your answers about Jurisdiction Choices
+subquestion: |-
+  ${ jurisdiction_choices.table }
+---
+event: |-
+  generic_report.revisit
+id: |-
+  revisit generic_report
+question: |-
+  Edit your answers about Generic Report
+subquestion: |-
+  ${ generic_report.table }
+---
+event: |-
+  placeholder_signature.revisit
+id: |-
+  revisit placeholder_signature
+question: |-
+  Edit your answers about Placeholder Signature
+subquestion: |-
+  ${ placeholder_signature.table }
+---
+event: |-
+  quality_check_overlay.revisit
+id: |-
+  revisit quality_check_overlay
+question: |-
+  Edit your answers about Quality Check Overlay
+subquestion: |-
+  ${ quality_check_overlay.table }
+---
+columns:
+  - 'Make this a purely informational screen': |-
+      row_item.is_informational_screen if hasattr(row_item, 'is_informational_screen') else ''
+  - 'What should the title of this screen be?': |-
+      row_item.question_text if hasattr(row_item, 'question_text') else ''
+  - 'What is the text that appears under the title?': |-
+      row_item.subquestion_text if hasattr(row_item, 'subquestion_text') else ''
+  - 'Select the fields that will appear on this screen': |-
+      row_item.field_list if hasattr(row_item, 'field_list') else ''
+  - 'Override the default logic flow by adding a "continue button field"': |-
+      row_item.has_mandatory_field if hasattr(row_item, 'has_mandatory_field') else ''
+  - 'Docassemble variable name, in `snake_case`': |-
+      row_item.variable if hasattr(row_item, 'variable') else ''
+  - 'On-screen label or prompt': |-
+      row_item.label if hasattr(row_item, 'label') else ''
+  - 'Field type': |-
+      row_item.field_type if hasattr(row_item, 'field_type') else ''
+  - 'Options (one per line)': |-
+      row_item.choices if hasattr(row_item, 'choices') else ''
+  - 'Screen title': |-
+      row_item.question_text if hasattr(row_item, 'question_text') else ''
+  - 'Text on your question screen': |-
+      row_item.subquestion_text if hasattr(row_item, 'subquestion_text') else ''
+  - fld_order_list: |-
+      row_item.fld_order_list if hasattr(row_item, 'fld_order_list') else ''
+  - table_data: |-
+      row_item.table_data if hasattr(row_item, 'table_data') else ''
+rows: |-
+  interview
+table: |-
+  interview.table
+---
+columns:
+  - 'Descriptive name': |-
+      row_item.description if hasattr(row_item, 'description') else ''
+  - 'Dependency to add to setup.py': |-
+      row_item.dependency_name if hasattr(row_item, 'dependency_name') else ''
+  - 'YAML file name to add to `include` statement (optional)': |-
+      row_item.include_name if hasattr(row_item, 'include_name') else ''
+  - 'Enable by default': |-
+      row_item.default if hasattr(row_item, 'default') else ''
+rows: |-
+  jurisdiction_choices
+table: |-

```

Comment:
Minor nit: I'd put the table attribute at the top of the block, since that's usually the most important piece of info about the block